pipelines/helfi-redirector-review.yml

```
@@ -0,0 +1,54 @@
+#
+# Review pipeline. Run build and deploy for Platta dev environments.
+# Pipeline runs different tests e.g. unittest and browser tests.
+#
+# Continuous integration (CI) triggers cause a pipeline to run whenever you push
+# an update to the specified branches or you push specified tags.
+# only PR trigger pipeline
+trigger: none
+
+# Pull request (PR) triggers cause a pipeline to run whenever a pull request is
+# opened with one of the specified target branches, or when updates are made to
+# such a pull request.
+#
+# GitHub creates a new ref when a pull request is created. The ref points to a
+# merge commit, which is the merged code between the source and target branches
+# of the pull request.
+#
+# Opt out of pull request validation
+pr:
+  # PR target branch
+  branches:
+    include:
+    - main
+  paths:
+    exclude:
+    - '*release-please*'
+    - '*.md'
+    - '.github/'
+    - 'pipelines/helfi-redirector-release.yml'
+    - 'pipelines/helfi-redirector-dev.yml'
+    - 'docker-compose*'
```

We use compose.yaml format for the compose file ([as recommended by Docker](https://docs.docker.com/compose/intro/compose-application-model/#the-compose-file))

```suggestion
    - 'compose*'
```

---

fixed